Description: Fixed wrong vboxvideo_drv_system_SOURCES reference.
 Even the comment says we need at least 17
 "Build using local X.Org headers.  We assume X.Org Server 1.7 or later."

Author: Felix Geyer <fgeyer@debian.org>

---

Origin: debian
Forwarded: https://www.virtualbox.org/ticket/12803
Last-Update: <2014-03-10>

--- virtualbox-4.3.8-dfsg.orig/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
+++ virtualbox-4.3.8-dfsg/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
@@ -376,7 +376,7 @@ ifdef VBOX_USE_SYSTEM_XORG_HEADERS
         /usr/include/libdrm \
         /usr/include/xorg \
         /usr/include/pixman-1
- vboxvideo_drv_system_SOURCES := $(vboxvideo_drv_15_SOURCES)
+ vboxvideo_drv_system_SOURCES := $(vboxvideo_drv_17_SOURCES)
 endif
 
 
